packages/SwingSet/src/kernel/state/storageWrapper.js

```diff
@@ -36,6 +38,7 @@ function* mergeSortedIterators(it1, it2) {
       v2 = it2.next();
       yield result;
     } else {
+      // TODO? assert(v1.value > v2.value);
```

Comment:
That could only fail if there's a NaN involved, yeah? Or if `.value` is an unstable getter, or something with an unstable `.toString`? Maybe just update the comment to indicate this is only meant for strings.